WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -1051,6 +1060,73 @@ class MySiteFragment : Fragment(),
         }
     }
 
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    fun onEventMainThread(event: StorySaveResult) {
+        EventBus.getDefault().removeStickyEvent(event)
+        if (event.isSuccess()) {
+            // TODO WPSTORIES add TRACKS
+            // AnalyticsTracker.track(Stat.MY_SITE_ICON_UPLOAD_UNSUCCESSFUL);
```

Comment:
This is a nitpick 😅 when I read this, I got a bit confused because I had to process that the event in `track` wasn't related to this code, which is expected since the `TODO` is above. So could we do without the `AnalyticsTracker` here? Also, I think we have a wrapper around the `AnalyticsTracker` as well to make classes more testable.

Reply:
:D I know what you mean - and this is all tackled in https://github.com/wordpress-mobile/WordPress-Android/pull/12074 👍

Reply:
Okay understood! As I said, I know I will comment on a few things that are fixed in future PRs, so bear with me 🙇